lldb/tools/lldb-dap/LLDBUtils.cpp

```diff
@@ -33,14 +39,49 @@ void RunLLDBCommands(llvm::StringRef prefix,
       const char *error = result.GetError();
       strm << error;
     }
+  };
+
+  lldb::SBCommandInterpreter interp = g_dap.debugger.GetCommandInterpreter();
+  for (llvm::StringRef command : commands) {
+    lldb::SBCommandReturnObject result;
+    bool quiet_on_success = false;
+    bool abort_on_error = false;
+
+    while (parse_command_directives) {
+      if (command.starts_with("?")) {
+        command = command.drop_front();
+        quiet_on_success = true;
+      } else if (command.starts_with("!")) {
+        command = command.drop_front();
+        abort_on_error = true;
+      } else {
+        break;
+      }
+    }
+
+    interp.HandleCommand(command.str().c_str(), result);
```

Comment:
If `parse_command_directives` is false and the commands start with `?` and `!` they'll fail right? Would you want the commands to run (ignoring the `?` and `!`) even though `parse_command_directives` is off?

Reply:
yes, that's to prevent someone unintentionally sending a command that starts with ! using the debug console and then having their debug session terminate